src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -338,7 +326,7 @@ public WebView(WebView webView) : this(Interop.WebView.new_WebView__SWIG_1(WebVi
         /// <summary>
         /// The url to load.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
         public string Url
```

Comment:
Can you please make it hidden again? We are planing to fix XXXUrl api to use System.Uri type.

Reply:
done.